optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/InnerConstraintFactory.java

```diff
@@ -96,15 +98,19 @@
                             ProblemFactCollectionProperty.class, entry.getKey());
                 });
         Set<Class> allAcceptedClassSet = concat(concat(entityClassStream, factClassStream), factCollectionClassStream)
-                .collect(Collectors.toSet());
+                .collect(toSet());
```

Comment:
I like this. It improves readability. I used to be afraid of static imports in the past because they make it difficult to guess where the method comes from but in some cases, like this one, they're not confusing at all. In the context of a stream it's always clear what `toSet()` or `toList()` means. I'm going to apply this style in my code as well.